### PR TITLE
Issue #1440 Simplifying conditions in pkg/util/filehelper

### PR DESCRIPTION
**Pull request by surajnarwade**

### CODE REVIEW

File: pkg/util/filehelper/file_test.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 
 func Test_non_existent_directory_returns_false(t *testing.T) {
 	path := filepath.Join("this", "path", "really", "should", "not", "exists", "unless", "you", "have", "a", "crazy", "setup")
-	if Exists(path) == true {
```
Comment (praveenkumar) [inline pkg/util/filehelper/file_test.go]: 👍

File: pkg/util/filehelper/file_test.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ func Test_existent_directory_returns_true(t *testing.T) {
 		t.Fatal("Unexpected error: " + err.Error())
 	}
 
-	if Exists(testDir) == false {
+	if !Exists(testDir) {
```
Comment (praveenkumar) [inline pkg/util/filehelper/file_test.go]: 👍

File: pkg/util/filehelper/file.go [hunk 3]
```diff
@@ -163,9 +163,6 @@ func IsEmptyDir(path string) bool {
 	defer f.Close()
 
 	_, err = f.Readdirnames(1)
-	if err == io.EOF {
-		return true
-	}
 
-	return false
+	return err == io.EOF
```
Comment (praveenkumar) [inline pkg/util/filehelper/file.go]: 👍

Comment (budhrg) [inline pkg/util/filehelper/file.go]: :+1:

**Comment by gbraad** [comment]
Please create an issue, as commit description misses "Issue #xxxx". Create a refactor task to deal with `filehelper`.

**Comment by budhrg** [comment]
@surajnarwade , I have created https://github.com/minishift/minishift/issues/1440 for this PR. 

Could you update your commit message as `Issue #1440 <message>` to align with our consistent commit messaging rule?

File: pkg/util/filehelper/file_test.go (hunk 1 above)
Comment (gbraad) [reply pkg/util/filehelper/file_test.go]: although identical in use, in some cases this construct is actually easier to reason about

File: pkg/util/filehelper/file.go (hunk 3 above)
Comment (gbraad) [reply pkg/util/filehelper/file.go]: 😆

**Comment by surajnarwade** [comment]
Hey guys @praveenkumar @budhrg @gbraad , thanks for reviewing,  updated description :)

**Comment by praveenkumar** [comment]
@surajnarwade you also need to update commit message which should have `Issue #1440 <commit-message>`.

**Comment by LalatenduMohanty** [comment]
retest this please

**Comment by gbraad** [comment]
Merged as b65660f5053513f41a2d62155a288a7f2f336786

**Comment by surajnarwade** [comment]
sorry my bad, I haven't seen it (merged) before sending

**Pull request closed without merging**